resdk/analysis/_register.py

```diff
@@ -0,0 +1,28 @@
+"""Patch ReSDK resources with analysis methods."""
+from __future__ import absolute_import, division, print_function, unicode_literals
+
+from resdk.resources import Collection, Relation, Sample
+
+from resdk.analysis.alignment import bowtie2, hisat2
+from resdk.analysis.chip_seq import macs, rose2
+from resdk.analysis.expressions import cuffnorm, cuffquant
+
+
+Collection.run_bowtie2 = bowtie2
```

Comment:
There must be a better way then `_register`. You do not have to find it now, but keep thinking about this.